src/c++/vdo/base/slab-depot.c

```diff
@@ -2288,14 +2297,21 @@ static void load_reference_block(struct vdo_waiter *waiter, void *context)
 static void load_reference_blocks(struct vdo_slab *slab)
 {
 	block_count_t i;
+	u64 step = slab->allocator->refcount_blocks_per_big_vio;
```

Comment:
It's a local, so not a big deal, but "step" feels awful as a variable name. "count" "block_count", "group_size"?